config/logstash.yml

```diff
@@ -91,7 +91,8 @@
 #
 # http.host: "127.0.0.1"
 #
-# Bind port for the metrics REST endpoint
+# Bind port for the metrics REST endpoint, this option also accept a range
+# (9000-10000) and logstash will pick up the first available ports.
```

Comment:
We should not allow `9000-10000`. That's a big range and can collide with ES and lots of other applications. How about `9600-9700` inclusive?